src/components/ChatLog.js

```diff
@@ -0,0 +1,34 @@
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+  return (
+    <div className="chatLog">
+      {props.entries.map((entry) => {
```

Comment:
While this works, I recommend moving this .map call outside of the return statement and storing the resulting array in a variable.  This will simplify the return statement, which can improve readability.